src/ast/spans.rs

```diff
@@ -342,6 +343,15 @@ impl Spanned for Statement {
                 copy_options: _,
                 validation_mode: _,
             } => Span::empty(),
+            Statement::CopyIntoSnowflakeLocation {
```

Comment:
`CopyIntoSnowflake` shares `into`, `partition`, `file_format`, `copy_options` fields  -- did you consider keeping `Statement::CopyIntoSnowflake` and add an enum like `target` to distinguish the two forms?